### ISSUE TITLE
New Feature: uf_plot_erpimage

### ISSUE THREAD

**Issue by behinger** [unfoldtoolbox/unfold#49]
I'm currently working on an erpimage function that allows to easily plot modeled / full / reduced-data

Currently I offer the following options (plus some non-essential ones)
```matlab
{'type' {'raw','deconv','nodeconv','residuals','fullmodel'}

'remove' % which event(s) predictor(s) pair(s) to remove 

remove-format: {'eventA',{'(Intercept)','stimA'}} (or a cell array of such cell arrays)


'keep'    {'eventA',{'stimB'}} (or a cell array of such cell arrays)

'alignto';  % what should be the event to align the erpimage?

'sort_align'   % what event should the erpimage be sorted to - default same as alignto

'sort_by''latency';% what field should it be sorted by (latency, amplitude, condition etc.)

'searchtime' % in what range should we search for the sort-event?
```
You can either specify ```cfg.keep```, to make a minimal set of which events + predictors to use for the model, or specify ```cfg.remove``` to use the full model and remove events or predictors. I hope this function helps people understand what the unfold toolbox is doing.

An example for a model stimonset: y~1,  saccade: y~1+spl(sac_amp,10)
Plot the stimonset modeled data & sort it according to the first following saccade latency
```matlab
        cfg = [];
        cfg.type = 'deconv';
        cfg.alignto = {'stimonset'};
        cfg.sort_align = 'saccade';
        cfg.sort_by = 'latency';
        
        cfg.channel = 32;
        cfg.searchtime = [100 1000];
        cfg.winrej = winrej;
        
        cfg.keep = { {{'stimonset'},{'2_(Intercept'}} }
       %alternative; equivalent in this case:
       %cfg.remove= { {{'saccade'},{'(Intercept)','sac_amplitude'}}
	uf_erpimage(EEG,cfg);
```

## IMPORTANT: COLOR SCALES ARE NOT IDENTICAL!
![grafik](https://user-images.githubusercontent.com/10183650/44431408-10cbaf00-a59e-11e8-85f2-51a4b414bf7e.png)

In comparison to see the "raw" erpimage you can simply state:

```cfg.type = 'raw'```
![grafik](https://user-images.githubusercontent.com/10183650/44431483-58523b00-a59e-11e8-92c3-91e5713b1013.png)

or depending on what you want

```cfg.type = 'fullmodel'```

![grafik](https://user-images.githubusercontent.com/10183650/44431506-6bfda180-a59e-11e8-8eeb-75410aa9f544.png)

Will probably add a tutorial for this one, as the possibilities are very large on how to use this function!

**Comment by behinger** [comment]
This function is much more involved to program and use properly. If you want to have an early access, you can send me an email.

**Comment by behinger** [comment]
the function has already been included in unfold 1.1
